### PR TITLE
fix: shift selection bug(#535)

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has description for the breaking change

### Description

#535 

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/dispatch/mouse.ts [hunk 1]
```diff
@@ -277,14 +275,8 @@ export function selectionUpdate(store: Store, dragStartData: DragData, dragData:
   endRowIndex = findRowIndexByPosition(store, viewInfo);
 
   if (curInputRange === null) {
```
Comment (seonim-ryu) [inline src/dispatch/mouse.ts]: `isNull()`로 변경하면 좋겠네요ㅎ

File: src/dispatch/mouse.ts [hunk 2]
```diff
@@ -277,14 +275,8 @@ export function selectionUpdate(store: Store, dragStartData: DragData, dragData:
...
-    const startViewInfo = {
-      pageX: dragStartData.pageX,
-      pageY: dragStartData.pageY,
-      scrollTop,
-      scrollLeft
-    };
-    startColumnIndex = findColumnIndexByPosition(store, startViewInfo);
-    startRowIndex = findRowIndexByPosition(store, startViewInfo);
+    startColumnIndex = store.focus.columnIndex!;
+    startRowIndex = store.focus.rowIndex!;
```
Comment (seonim-ryu) [inline src/dispatch/mouse.ts]: 요렇게 구조분해하는건 어떠신지요ㅎ
```js
const { columnIndex, rowIndex } = store.focus;
startColumnIndex = columnIndex!;
startRowIndex = rowIndex!;
```

File: src/store/selection.ts [hunk 3]
```diff
@@ -149,6 +149,21 @@ export function create({
         L: leftSideStyles,
         R: rightSideStyles
       };
+    },
+
+    get rangeWithRowHeader(this: Selection) {
```
Comment (seonim-ryu) [inline src/store/selection.ts]: 아주아주아주 사소한거지만..`rowHeader` 여러 개가 생성될 수 있으므로.. 복수형이 되어야하지 않을까요?ㅎ
`rangeWithRowHeaders`

**Comment by seonim-ryu** [comment]
[7/18] 리뷰 완료! 오늘도 짧고 굵게ㅎ 고생하셨습니다~

File: src/dispatch/mouse.ts (hunk 2 above)
Comment (js87zz) [reply src/dispatch/mouse.ts]: 이 부분은 변경되면 안될 것 같습니다. frozen coulmn이 있는 경우는 제대로 동작을 안할 것 같네요.

File: src/store/selection.ts (hunk 3 above)
Comment (jung-han) [reply src/store/selection.ts]: `visibleColumnsWithRowHeader` 에 맞춰서 수정하긴 했는데..전부수정해야할까요?

**Pull request merged**